mne/annotations.py

```diff
@@ -85,6 +85,14 @@ def __init__(self, onset, duration, description,
         self.duration = duration
         self.description = np.array(description, dtype=str)
 
+    def __repr__(self):
+        """Show the representation."""
+        kinds = sorted(set(['"%s"' % d.split(' ')[0]
+                            for d in self.description]))
+        kinds = ', '.join(kinds[:3]) + ('' if len(kinds) <= 3 else '...')
```

Comment:
We don't need to support Python 2.6 any longer. please remove list comprehension and use set comprehension directly.